onepassword/client.py

```diff
@@ -330,9 +330,9 @@ def get_item(uuid, fields=None):
 
         """
         if isinstance(fields, list):
-            item = json.loads(read_bash_return("op get item {} --fields {}".format(uuid, ",".join(fields))))
+            item = json.loads(read_bash_return("op get item {} --fields \"{}\"".format(uuid, ",".join(fields))))
         elif isinstance(fields, str):
-            item = {fields: read_bash_return("op get item {} --fields {}".format(uuid, fields))}
+            item = {fields: read_bash_return("op get item {} --fields \"{}\"".format(uuid, fields))}
```

Comment:
Apologies for slow response here. Only a small suggestion:
```suggestion
            item = {fields: read_bash_return('op get item {} --fields \"{}\"'.format(uuid, fields))}
```
To make sure no confusion with single / double quotations and please test before committing :) we will also test on our side before merge of course!